ivadomed/main.py

```diff
@@ -282,10 +282,12 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
         # PREDICTION FOLDER
         path_preds = os.path.join(log_directory, 'pred_masks')
         # If the prediction folder does not exist, run Inference first
-        if not os.path.isdir(path_preds):
-            print('\nRun Inference\n')
-            context["command"] = "test"
-            run_command(context)
+        if os.path.isdir(path_preds):
+            context['testing_parameters']['uncertainty']['n_it'] = 0
```

Comment:
Re-run prediction since the _pred.nii.gz file is the prediction with previous post-processing step (not raw prediction), but MC iterations are not re-run.